docs/content/topics/operators/gateways.md

```diff
@@ -11,340 +11,421 @@ aliases:
+a `brig event create` command, `brig` creates a Brigade event of source
+`brigade.sh/cli` and of type `exec`. Brigade itself processes this event no
+differently than it processes events from gateways tracking external activity.
 
 There are no rules about what can be used as a trigger for an event. One could
-write a gateway that listens on a message queue, or runs as a chat bot, or watches
-files on a filesystem... any of these could be used to trigger a new Brigade event.
+write a gateway that listens on a message queue, or runs as a chat bot, or
+watches files on a filesystem... any of these could be used to trigger a new
+Brigade event.
 
-The remainder of this guide explains how gateways work and how you can create custom
-gateways.
+The remainder of this guide explains how gateways work and how you can create
+custom gateways.
+
+[Helm]: https://helm.sh
+
+## Available Gateways
 
-## An Event Is A Secret
+Currently, the list of official Brigade v2 gateways that process external
+events is as follows:
 
-The most important thing to understand about a Brigade event is that it is simply
-a [Kubernetes Secret](https://kubernetes.io/docs/concepts/configuration/secret/)
-with special labels and data.
+* [Github Gateway](https://github.com/brigadecore/brigade-github-gateway)
+* [BitBucket Gateway](https://github.com/brigadecore/brigade-bitbucket-gateway/tree/v2)
+* [CloudEvents Gateway](https://github.com/brigadecore/brigade-cloudevents-gateway)
+* [Docker Hub Gateway](https://github.com/brigadecore/brigade-dockerhub-gateway)
+* [Azure Container Registry Gateway](https://github.com/brigadecore/brigade-acr-gateway)
 
-When a new and appropriately labeled secret is created in Kubernetes, the Brigade
-controller will read that secret and start a new Brigade worker to handle the event.
-Secrets have several characteristics that make them a great fit for this role:
+Follow the installation instructions provided in each gateway's repository to
+learn how to get started.
 
-- They are designed to protect data (and we expect them to mature in this capacity)
-- They can be mounted as volumes and environment variables.
-- The payload of a secret is flexible
-- Secrets have been a stable part of the Kubernetes ecosystem since Kubernetes 1.2
+### The Anatomy of a Brigade Event
 
-Because of these features, the Brigade system uses secrets for bearing event information.
+All gateways perform the same job, that is, to translate activity and context
+from some source into a Brigade event. Let's now look at the structure of a
+Brigade event.
 
-### The Anatomy of a Brigade Event Secret
+A Brigade Event is defined primarily by its source and type values, worker
+configuration and worker status.
 
-Here is the structure of a Brigade event secret. It is annotated to explain what
-data belongs in what fields.
+Here is a YAML representation of an event created via the `brig event create`
+command for the [01-hello-world sample project].
```

Comment:
I wonder if we should explicitly state that many of the fields shown are system-generated.

Or maybe we can present an abridged YAML representation (and explicitly acknowledge that it is abridged) that focused only on the fields that can be set by a client (gateway in this case).

Reply:
Added additional text around this... WDYT?